### ISSUE TITLE
new nunjucks templates structure

### ISSUE THREAD

**Issue by ronaldruzicka** [actum/gulp-dev-stack#146]
I'd like to introduce new structure for nunjucks templates.

For now, we usually have **base(atoms)**, **components**, and pages are generated from any `.nunj` in **tpl** folder. 
So, e.g. if you want to see the component you are working on, you also have to create a page which has this component. Because `.nunj` files from other folders that are inside **tpl** are not being converted to `.html`.
What if we compiled all of the actual nunjucks templates and show them in `index.nunj` file like: 
**atoms**:
- buttons
- form
- headings

**components**:
- header
- slider
- login-form

**pages**:
- homepage
- contact
- products

So that we could see all the parts in the `index.nunj`. What do you think?

**Comment by kettanaito** [comment]
👍 I would go for this if we can list it in `index.nunj` dynamically.

**Comment by vbulant** [comment]
I’ve added on hold label coz I think we should discuss #163 first. In case we go for it, this issue is not relevant anymore or at least it should be implemented after #163.
